src/testUtils/java/uk/gov/hmcts/reform/wataskmanagementapi/config/GivensBuilder.java

```diff
@@ -102,6 +102,20 @@ public GivensBuilder iAddVariablesToTaskWithId(String taskId, CamundaProcessVari
         return this;
     }
 
+    public GivensBuilder iAddLocalVariablesToTaskWithId(String taskId, CamundaProcessVariables processVariables) {
+        given()
+            .contentType(APPLICATION_JSON_VALUE)
+            .baseUri(camundaUrl)
+            .body(new Modifications(processVariables.getProcessVariablesMap()))
+            .when()
+            .post("/task/{task-id}/variables", taskId)
```

Comment:
I think you should use /task/{task-id}/localVariables. I think the documentation for /task/{task-id}/variables is a bit vague when dealing with variables that do not already exist. As we know these should be task local variables better to use the /localVariables endpoint. Then we can be sure we are setting a task local variable not a process variable.